### PR TITLE
Code quality/no spaces inside parenthesis

### PR DESCRIPTION
**Pull request by phil-davis**
## Description
Add ``php_cs_fixer`` rule  ``no_spaces_inside_parenthesis``

## Motivation and Context
``combine_consecutive_issets`` in #31271 leaves behind white space.

## How Has This Been Tested?
Run ``php_cs_fixer`` locally

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:
- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Comment by phil-davis** [comment]
I put this on top of #31271 as they "sort of go together" but "not really". 

IMO this seems like "a good thing" to tidy up some other random spaces around the code base.

**Comment by phil-davis** [comment]
Why do I get no codecov reported?

File: apps/dav/tests/unit/Connector/Sabre/FileTest.php [hunk 1]
```diff
@@ -71,9 +71,9 @@ class FileTest extends TestCase {
 
 	public function setUp() {
 		parent::setUp();
-		unset($_SERVER['HTTP_OC_CHUNKED']);
-		unset($_SERVER['CONTENT_LENGTH']);
-		unset($_SERVER['REQUEST_METHOD']);
+		unset($_SERVER['HTTP_OC_CHUNKED'], $_SERVER['CONTENT_LENGTH'], $_SERVER['REQUEST_METHOD']);
+
```
Comment (DeepDiver1975) [inline apps/dav/tests/unit/Connector/Sabre/FileTest.php]: still stupid blank lines ....

Comment (phil-davis) [reply apps/dav/tests/unit/Connector/Sabre/FileTest.php]: https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/2804

``no_extra_consecutive_blank_lines`` looks a good thing to add...

**Comment by phil-davis** [comment]
Now I have 589 files changed. Can we get it over 1000?

**Comment by phil-davis** [comment]
I lost code coverage by deleting blank lines???
Poor codecov - it does not really know what to do with this sort of rubbish change.

File: lib/private/Files/Cache/Scanner.php [hunk 2]
```diff
@@ -189,7 +189,7 @@ public function scanFile($file, $reuseExisting = 0, $parentId = -1, $cacheData =
 						$fileId = $cacheData['fileid'];
 						$data['fileid'] = $fileId;
 						// only reuse data if the file hasn't explicitly changed
-						if (isset($data['storage_mtime']) && isset($cacheData['storage_mtime']) && $data['storage_mtime'] === $cacheData['storage_mtime']) {
+						if (isset($data['storage_mtime'], $cacheData['storage_mtime'])   && $data['storage_mtime'] === $cacheData['storage_mtime']) {
```
Comment (phil-davis) [inline lib/private/Files/Cache/Scanner.php]: and here it puts in an extra space char - to fix...

File: lib/private/Files/Cache/Updater.php [hunk 3]
```diff
@@ -123,7 +123,7 @@ public function update($path, $time = null) {
 
 		$data = $this->scanner->scan($path, Scanner::SCAN_SHALLOW, -1, false);
 		if (
-			isset($data['oldSize']) && isset($data['size']) &&
+			isset($data['oldSize'], $data['size'])   &&
```
Comment (phil-davis) [inline lib/private/Files/Cache/Updater.php]: too much space here

File: lib/private/NaturalSort.php [hunk 4]
```diff
@@ -110,7 +110,7 @@ public function compare($a, $b) {
 		$aa = self::naturalSortChunkify($a);
 		$bb = self::naturalSortChunkify($b);
 
-		for ($x = 0; isset($aa[$x]) && isset($bb[$x]); $x++) {
+		for ($x = 0; isset($aa[$x], $bb[$x])  ; $x++) {
```
Comment (phil-davis) [inline lib/private/NaturalSort.php]: B\onus space here

**Comment by phil-davis** [comment]
@DeepDiver1975 this is ready for review and merge, if you want to take the plunge...

**Comment by patrickjahns** [comment]
Can we reset codecov results ? If not, can we put it into a clean PR so we know what codecov finds issues with

**Comment by phil-davis** [comment]
PR #31306 has a clean cherry-pick of all these commits. Let's see what CI thinks about it.

**Comment by phil-davis** [comment]
The other PR also gets 58.51% of diff hit (target 62.58%). I guess that of the lines touched by this code format change, on average they are only inside test-covered code 58.51% of the time, a bit less than the repo average of 62.58%.

I don't think we require to do massive unit test writing in order to implement code-style standards???

**Comment by phil-davis** [comment]
Are we going to backport these code-style changes?

I would like that to happen, to minimize future rubbish conflicts when backporting other PRs.

**Comment by DeepDiver1975** [comment]
> Are we going to backport these code-style changes?

yes - we will - but I'd first like to get the style sorted out on master and let devs adopt and see if this all works out

**Comment by phil-davis** [comment]
sounds good. "backporting" in the end will be adding the "final" ``.php_cs`` and running it to fix everything. (i.e. a 1-by-1 backport of each PR in ``master`` will not be particularly easy and will not necessarily get things passing in ``stable10`` anyway)

**Comment by phil-davis** [comment]
Someone override codecov and merge if you like it.

**Review by DeepDiver1975 — APPROVED** [review]

**Comment by phil-davis** [comment]
Effective backport is included in #31453

**Pull request merged**